greeting-service/src/main/java/io/openshift/booster/service/NameCacheUtil.java

```diff
@@ -29,22 +32,30 @@
 @Component
 public class NameCacheUtil {
 
-    private final Cache springCache;
+    private static final String KEY = "key";
+
+    private final int ttlMillis;
     private final BasicCache<String, String> infispanCache;
```

Comment:
I just noticed -- this variable is called "infispan", should be "infinispan" :-)

Reply:
Haha :P Damn typos :). I'll fix it